sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDirectorySteps.java

```diff
@@ -83,16 +90,22 @@ public CaseDirectorySteps(
           TimeUnit.SECONDS.sleep(3); // needed for table refresh
         });
 
+    And(
+        "I filter by Random CaseID on Case directory page",
```

Comment:
the random approach won't be stable, because once it might find a valid one. Let's try to use an uid lix aaaaa-aaaa-bbb etc so this will always return 0 results, and rename the method